example/example.c

```diff
@@ -88,7 +95,7 @@ int main(void) {
      */
     sirfileid fileid = sir_addfile("libsir-example.log", SIRL_ALL, SIRO_NONAME | SIRO_NOHOST);
     if (0 == fileid)
-        report_error();
+        (void)report_error();
```

Comment:
At this point with this many (void) in front of it, might as well nuke the return type and just not use it as the return value.